gateway-service-definitions/src/main/resources/services/yarnui/2.7.0/rewrite.xml

```diff
@@ -219,6 +218,11 @@
         <apply path="https?://[^/':,]+:[\d][^']*" rule="YARNUI/yarn/outbound/node2"/>
     </content>
 </filter>
+<filter name="YARNUI/yarn/outbound/filter/cluster1">
```

Comment:
So, how is this filter called? I don't see any matching route for `YARNUI/yarn/outbound/filter/cluster1` in service.xml

Reply:
That's probably why it works for me then. :-)

Reply:
@moresandeep going to try altering the path for the `YARNUI/yarn/outbound/apps/cluster1` then.  Currently rebuilding and deploying locally.

Reply:
> @moresandeep going to try altering the path for the `YARNUI/yarn/outbound/apps/cluster1` then. Currently rebuilding and deploying locally.

So I see no problems yet running with this patch.  The Kill button works, and everything else seems functional (except for the history url noted in KNOX-2528/KNOX-1032).

If I understand the rules and patterns correctly, I can't imagine anything should really break as a result of this, because `/ws/v1/.*` catches and handles `/ws/v1/cluster/apps/application` correctly.